docs/integrations/gitlab/org.md

```diff
@@ -58,14 +58,16 @@ amount of data, this can take significant time and resources.
 The token used must have the `read_api` scope, and the Users and Groups fetched
 will be those visible to the account which provisioned the token.
 
+**NOTE**: If any groups that are being ingested are empty groups and the user which provisioned the token is shared with a higher level group via [group sharing](https://docs.gitlab.com/ee/user/group/manage.html#share-a-group-with-another-group) and you don't see the expected number of `Group` entities in the catalog you may be hitting this [Gitlab issue](https://gitlab.com/gitlab-org/gitlab/-/issues/267996).
+
 ```yaml
 catalog:
   providers:
     gitlab:
       yourProviderId:
         host: gitlab.com
         orgEnabled: true
-        group: org/teams # Optional. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
+        group: org/teams # Required for gitlab.com. Optional for self managed. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
```

Comment:
Probably want to describe the effect on user ingestion in the gitlab.com case here too, something like "For gitlab.com, accepts only users _associated with_ the provided group's path"... I don't like the "associated with" term, I'm just not sure what the right precise terminology is.

Reply:
Took another stab and fleshing out the GitLab Org Data docs